notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -22,7 +22,7 @@
   },
```

Comment:
<p>Here, if you could add some description and inference from the above graph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/289/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>can you also add the units for the axes where applicable?</p>

Reply:
<p>Can this also all just be one plot, using <code>plt.subplots(2, 3)</code>  so you have 2 rows of plots since its still all one idea being presented here really. </p>